lib/sufia/engine.rb

```diff
@@ -72,6 +73,7 @@ class Engine < ::Rails::Engine
     config.temp_file_base = nil
     config.redis_namespace = "sufia"
     config.fits_path = "fits.sh"
+    config.libreoffice_path = "/opt/local/libreoffice_path/bin/soffice"
```

Comment:
default should probably be `soffice` which requires it to be on your path. This will work for the largest number of systems.